spec/models/like_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should belong_to(:post) }
+  end
+end
```

Comment:
- [ ] Also here, Just as mentioned earlier, there should be a test that tests the `update_post_likes_counter` method. Please add these tests.